pkgs/build-support/writers/default.nix

```diff
@@ -207,7 +207,7 @@ rec {
   } /* sh */ ''
     # nginx-config-formatter has an error - https://github.com/1connect/nginx-config-formatter/issues/16
     awk -f ${awkFormatNginx} "$textPath" | sed '/^\s*$/d' > $out
-    gixy $out
+    gixy --skips host_spoofing $out
```

Comment:
Typically you want this though, it just happens that we have a configuration which has no choice but to use that particular kind of NGINX configuration...